pkg/osquery/tables/pwpolicy/pwpolicy.go

```diff
@@ -0,0 +1,143 @@
+//+build darwin
+
+// Package pwpolicy provides a table wrapper around the `pwpolicy` macOS
+// command.
+//
+// As the returned data is a complex nested plist, this uses the
+// dataflatten tooling. (See
+// https://godoc.org/github.com/kolide/launcher/pkg/dataflatten)
+
+package pwpolicy
+
+import (
+	"bytes"
+	"context"
+	"os/exec"
+	"strings"
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+	"github.com/pkg/errors"
+)
+
+const PwpolicyPath = "/usr/bin/pwpolicy"
+const PwpolicyCmd = "getaccountpolicies"
```

Comment:
```suggestion
const pwpolicyPath = "/usr/bin/pwpolicy"
const pwpolicyCmd = "getaccountpolicies"
```

These don't need to be exported variables